ra/ra.go

```diff
@@ -1159,6 +1159,20 @@ func (ra *RegistrationAuthorityImpl) validateFinalizeRequest(
 	return csr, nil
 }
 
+func (ra *RegistrationAuthorityImpl) GetSCTs(ctx context.Context, sctRequest *rapb.SCTRequest) (*rapb.SCTResponse, error) {
+	cert, err := x509.ParseCertificate(sctRequest.PrecertDER)
+	if err != nil {
+		return nil, err
```

Comment:
Generally this error should be wrapped, because it is coming from an external package.